modules/nested-elements/assets/js/editor/nested-repeater/hooks/data/document/elements/create/nested-repeater-adjust-container-titles.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * For each nested repeater container item, adjust the container title,
+ * according to the nested repeater title, the result will be 'Tab #1', 'Tab #2' and so on instead of 'Container'.
+ * '_title' is used in navigator.
```

Comment:
```suggestion
 * On each nested widget creation, for each nested repeater container item, adjust the container title,
 * according to the nested repeater title, the result will be 'Tab #1', 'Tab #2' and so on instead of 'Container'.
 * '_title' is used in navigator.
```